### ISSUE TITLE
[P0] [BUG] Link in Control Plane app to Alerts-UI has the wrong IP-PORT

### ISSUE THREAD

**Issue by CESARDELATORRE** [Azure/mec-app-solution-accelerator#145]
[P0] [BUG] Link en Control Plan app to Alerts-UI has wrong PORT

The root cause can be that it's using a different IP address, why?
Previously, we were using the same IP address for all the services (the real IP of the machine) and then we just use different ports for the app. THat's the way it needs to be so we can also access the app from remote machines.
We need to use the real machine's IP.

<img width="886" alt="image" src="https://github.com/Azure/mec-app-solution-accelerator/assets/1712635/43dcbde6-2fe0-458e-bc2f-c49cfe356f50">

It goes to: 
 http://192.168.0.6:88/  --> Tha's not the IP

It should be, if I check the URL in K8s Dashboard:
http://192.168.0.4:88/

In the K8s dashboard services I see it like this:

<img width="631" alt="image" src="https://github.com/Azure/mec-app-solution-accelerator/assets/1712635/cd0fa015-2afa-484f-b615-6b8cc7e87dbf">

<img width="629" alt="image" src="https://github.com/Azure/mec-app-solution-accelerator/assets/1712635/239debf4-94fe-42cd-9af3-b3ea97ba506b">

Basically the link needs to find out the IP and port dynamically....

BUT MOST IMPORTANTLY, IT HAS TO USE THEREAL IP OF THE MACHINE, so you can access remotely easier.

Previosly (v 1.8) I think we were using the same IP for all the public services in K8s, as you can see here, the IP 172.30.1.75 for all the public services. But that's not the way I see it in v1.9 with Aio...:

![image](https://github.com/Azure/mec-app-solution-accelerator/assets/1712635/4bdc8078-b1d1-4915-930d-77036a149850)

**Comment by virubiotor** [comment]
@CESARDELATORRE 
Right now the code from the control-plane-ui-service uses the same protocol and IP that the control-plane-app and the 88 port

`function useAlertsDashboardUri() {
    const [alertsUiUri, setAlertsUiUri] = useState('');
    useEffect(() => {
        const newUri = `${window.location.protocol}//${window.location.hostname}:88`;
        setAlertsUiUri(newUri);
    }, []);

    return alertsUiUri;
}`

Previously (~3 months ago), it used to be an env variable with the address of the Alerts-UI service. I feel like this might be the change you are mentioning.
This change was made in this commit: https://github.com/Azure/mec-app-solution-accelerator/commit/f7c60f5797711c20e8575479c6e6528ef9031baa
Related issue: https://github.com/Azure/mec-app-solution-accelerator/issues/117#issuecomment-1954921164
![image](https://github.com/Azure/mec-app-solution-accelerator/assets/10944238/f31eb1fe-8fbf-4800-b413-7bc8096d0996)

Should be rollback this changes and use the Cluster IP as an environment variable that must be set by the user?

**Comment by CESARDELATORRE** [comment]
The important point is to be able to easily access to the UI apps from remote machines. Usually that's easier if using the machine's IP instead of new IPs, because we should use IPs for the UI apps that have to be in the same IP network where the physical machine is.

For instance, my machine's IP address is: 192.168.1.233

Wireless LAN adapter Wi-Fi:

   Connection-specific DNS Suffix  . : attlocal.net
   IPv6 Address. . . . . . . . . . . : 2600:1700:1990:9b40::b
   IPv6 Address. . . . . . . . . . . : 2600:1700:1990:9b40:5b22:4b05:e8a3:51a5
   Temporary IPv6 Address. . . . . . : 2600:1700:1990:9b40:6d86:56f9:64ea:5ad8
   Link-local IPv6 Address . . . . . : fe80::d44f:6c6b:da44:9f0e%17
   IPv4 Address. . . . . . . . . . . : **192.168.1.233**
   Subnet Mask . . . . . . . . . . . : 255.255.255.0
   Default Gateway . . . . . . . . . : fe80::3aa0:67ff:fe89:ebb2%17
                                       192.168.1.254

But then, the IPs for the control plane app and alerts-UI-dashboard are now:
http://192.168.0.6:90/
http://192.168.0.4:88/

Can a remote machine in my IP server/machine's IP network access those other IPs which are in a different IP network without doing additional complicated configurations? (i.e. Firewall, Router routes, etc.?)

The important thing is to easily access to the UI apps from remote machines.

**Comment by CESARDELATORRE** [comment]
Fixed, closing issue.
